src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -130,7 +130,7 @@ internal ProjectFileInfo(string filePath)
             }
 
             var projectInstance = project.CreateProjectInstance();
-            var buildResult = projectInstance.Build(TargetNames.Compile,
+            var buildResult = projectInstance.Build(new string[] { TargetNames.Compile, TargetNames.CoreCompile },
```

Comment:
Is this going to result in unintended behavior changes in other projects?

Reply:
It shouldn't. This is also what we do in Visual Studio.